.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: objective-c
-osx_image: xcode7
+osx_image: xcode8
```

Comment:
Yeah i don't think it changed anything that wasn't already a little crazy, which I think we tried not to be here.